tasties_app/templates/tasties_app/single_recipe.html

```diff
@@ -0,0 +1,58 @@
+{% extends 'tasties_app/base.html' %}
+{% load static %}
+{% block css %}
+    {% load bootstrap_icons %}
+    <link rel="stylesheet" type="text/css" href="{% static 'css/single_recipe.css' %}" />
+{% endblock %}
+{% block content %}
+    <div class="recipe-body">
+        <div class="recipe-header">
+            <h1>{{ recipe.title }}</h1>
+            <h2>{{ recipe.description }}</h2>
+            <div class="recipe-header-info">
+                <span class="author-name">by @{{ recipe.author_id }}</span>
+                <span class="recipe-date">{{ recipe.publication_date }}</span>
```

Comment:
I would add "Published on: " before the publication date.

Reply:
Liked it better without it, the same as in the example you sent me.